### PR TITLE
packaging: add rsyslog as dependency to packages

### PR DESCRIPTION
**Pull request by braydonk**
## Description
This PR adds rsyslog as package dependencies to both deb and rpm packages.

## Related issue
[b/339061751](http://b/339061751)

## How has this been tested?
Local docker build successful, but the real test is the integration tests in this PR.

## Checklist:
- Unit tests
  - [x] Unit tests do not apply.
  - [ ] Unit tests have been added/modified and passed for this PR.
- Integration tests
  - [x] Integration tests do not apply.
  - [ ] Integration tests have been added/modified and passed for this PR.
- Documentation
  - [x] This PR introduces no user visible changes.
  - [ ] This PR introduces user visible changes and the corresponding documentation change has been made.
- Minor version bump
  - [x] This PR introduces no new features.
  - [ ] This PR introduces new features, and there is a separate PR to bump the [minor version](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/VERSION) since the last [release](https://github.com/GoogleCloudPlatform/ops-agent/releases) already.
  - [ ] This PR bumps the version.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -7,7 +7,9 @@ Standards-Version: 3.9.4
 
 Package: google-cloud-ops-agent
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         rsyslog | system-log-daemon
```
Comment (igorpeshansky) [inline debian/control:12]: Explicit dependencies generally appear first, don't they? @quentinmit could confirm.

Comment (braydonk) [reply debian/control]: I think you're right. Fixed in [43aedd1](https://github.com/GoogleCloudPlatform/ops-agent/pull/1696/commits/43aedd190a82f3218406bb9b6b7e408659fce333)

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Comment by braydonk** [comment]
Merging because everything has passed before and the package files haven't changed since.

**Pull request merged**